pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -19,9 +23,12 @@ let
     # should contain all args but the binary. Can be either a string or list
     , wrapperArgs ? []
     # a limited RC script used only to generate the manifest for remote plugins
-    , manifestRc ? null
+    # , manifestRc ? null
```

Comment:
Was something meant to be noted here for commenting this out?

Reply:
ty for spotting this, I removed it, it's generated a bit later in the wrapper.